pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "envattrs"
+version = "19.1"
+description = "Populate attr.s or dataclass objects from environment variables"
+authors = ["Jonas Obrist <jonas.obrist@hennge.com>"]
```

Comment:
I wonder if Iskandar also had a hand in this

Reply:
https://github.com/HENNGE/envattrs/graphs/contributors

Reply:
I guess I'll keep it as just jonas for now, or would you like to be added in as well @dimaqq ?

Reply:
I have no authorship in this respected library 🙇🏻